src/openboardview/win32.cpp

```diff
@@ -158,6 +159,26 @@ const std::string get_user_dir(const UserDir userdir) {
 	return configPath;
 }
 
+// Case insensitive lookup of a filename at the given path
```

Comment:
Why do we need case-insensitive on win32? Isn't it by default?

Reply:
It can be made case-insensitive. But IIRC I mostly tried to add it to keep the same interface and behaviour than on Linux and macOS, as it was already used before on all platforms but with the POSIX interface (that's not really liked by MSVC).